README.md

```diff
@@ -378,7 +378,7 @@ See the release section on GitHub.
         * Binaries are signed with the GPG key
           [`B9E36A7275FC61B464B67907E06FE8F53CDC6A4C`](https://keyserver.ubuntu.com/pks/lookup?search=B9E36A7275FC61B464B67907E06FE8F53CDC6A4C&fingerprint=on&op=index).
     * or, using Winget, type `winget install Martchus.syncthingtray` in a Command Prompt window.
-    * or, using [Scoop](https://scoop.sh), type `scoop add bucket extras & scoop install extras/syncthingtray`.
+    * or, using [Scoop](https://scoop.sh), type `scoop bucket add extras & scoop install extras/syncthingtray`.
```

Comment:
By the way, is the single `&` ok? In "normal" shells it would put the job in the background.

Reply:
I know. In Windows, it's the command terminator, as `;` is in Bash.

Ironically, `&&` and `||` work as expected.